skimage/util/tests/test_crop.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+from skimage import data
+from skimage.util.crop import crop
+
+
+def test_2d_crop_1():
+    data = np.random.random((50, 50))
+    out_data = crop(data, [(0, 25)])
+    assert out_data.shape == (25, 50)
+
+
+def test_2d_crop_2():
+    data = np.random.random((50, 50))
+    out_data = crop(data, [(0, 25)], axis=[1])
+    assert out_data.shape == (50, 25)
+
+
+def test_2d_crop_3():
+    data = np.random.random((50, 50))
+    out_data = crop(data, [(0, 25), (0, 30)], axis=[1, 0])
+    assert out_data.shape == (30, 25)
```

Comment:
```suggestion
    assert np.testing.assert_array_equal(out_data, out_data[:25, :30])
```
For the tests we may as well test the actual values rather than just the shape